app/lib/find_constraint.rb

```diff
@@ -2,6 +2,6 @@
 
 class FindConstraint
   def matches?(request)
-    Settings.find_url&.include?(request.host) || request.host.include?('find-pr')
+    Settings.find_url&.include?(request.host) || request.host.include?('find-pr') || request.host.include?('find-review')
```

Comment:
```suggestion
    Settings.find_url&.include?(request.host) || request.host.include?('find-review')
```

If we replacing `cf` then we can remove `find-pr`?

Reply:
We can remove this when PaaS is decommissioned, but we need both while we dual run.